### PR TITLE
Deja Flynn 

### PR DESCRIPTION
**Pull request by DejaLFlynn**
Loop answers

### CODE REVIEW

**Comment by DejaLFlynn** [comment]
revised, I thought I pushed the correct bonus

File: loops_lab.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+for (let number = 0; number <= 10; number += 1) {
+  if (number % 2 !== 0) {
+    console.log(number + ' num is odd');
+  } else {
+    console.log(number + ' num is even');
+  }
+}
+
+let bynine = 99;
```
Comment (jungraejang) [inline loops_lab.js]: This is not exactly what the question was asking but good job with printing out whether the number is divisible by 9 or not.

File: loops_lab.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+while (bynine > 0) {
+  if (bynine % 9 !== 0) {
+    console.log(bynine  + ' is not divisible by 9');
+  } else {
+    console.log(bynine + 'is divisible by 9');
+  } bynine -= 1;
+}
+
+for (let i = 5; i <= 100; i += 10) {
```
Comment (jungraejang) [inline loops_lab.js]: Interesting way of solving this problem but can you think of a solution where you can use modulo operator to solve the same problem?

File: loops_lab.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  console.log(i);
+}
+
+//5.It would run infinite
+
+let str1 = '';
```
Comment (jungraejang) [inline loops_lab.js]: Code should like this: 
1
2
Fizz
4
Buzz
........
14
FizzBuzz
16

Your code will give something completely different. Next time, please read the question carefully and run the code before submitting it.

File: loops_lab.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    }
+  } else {
+    str1 += 'BUZZ'; {
+      console.log(str1);
+    }
+  }
+}
+
+//
+let str2 = '';
```
Comment (jungraejang) [inline loops_lab.js]: This code is not printing out what the question asked to print out. Try this problem again, and if you have any question, ask me. You got the right idea of using loop but logic isn't quite right inside the loop.

File: loops_lab.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (jungraejang) [inline loops_lab.js]: I think you understand how to use/create loops to solve problems, and also have good understanding of how loops work, but dealing with logics inside loops remains matter of further practice. Please read questions carefully next time, and also please reach out to me or Alejo if you have any questions or clarifications. It is important to get good basic understanding of Javascript in early stage. Keep practicing and keep up the good work!

**Pull request closed without merging**